.babelrc

```diff
@@ -3,6 +3,12 @@
   "env": {
     "test": {
       "presets": ["@babel/preset-env"]
+    },
+    "production": {
+      "presets": ["@babel/preset-env", "minify"],
```

Comment:
Doet ie nu ook al een transpile? Ik denk het niet? Zo niet is het nu ook even goed maar dan moeten we wellicht even een taakje aanmaken / er even over nadenken..